tests/foreman/maintain/test_upgrade.py

```diff
@@ -83,6 +95,70 @@ def test_positive_repositories_validate(sat_maintain):
     assert skip_message in result.stdout
 
 
+@pytest.mark.no_containers
+@pytest.mark.parametrize(
+    'custom_host',
+    [
+        {'deploy_rhel_version': '8', 'deploy_flavor': 'satqe-ssd.disk.xxxl'},
+        {'deploy_rhel_version': '8', 'deploy_flavor': 'satqe-ssd.standard.std'},
+    ],
+    ids=['default', 'medium'],
+    indirect=True,
+)
+def test_negative_pre_upgrade_tuning_profile_check(request, custom_host):
+    """Negative test that verifies a satellite with less than
+    tuning profile hardware requirements fails on pre-upgrade check.
+
```

Comment:
I would appreciate a list of hw requirements and requirements that are not met in each of the flavors. I think it could be useful for future debugging when requirements or flavor configurations are changed.

Reply:
I can add the specifications for each flavor to the docstring here, although we do have a list that can be shared if you would prefer that. I'm not exactly sure if we would ever change the flavors? Maybe @ogajduse or @bherrin3 could give more information on that?